services/headless-lms/server/src/programs/seed/seed_courses/seed_graded.rs

```diff
@@ -0,0 +1,179 @@
+use crate::programs::seed::builder::certificate::CertificateBuilder;
+use crate::programs::seed::builder::chapter::ChapterBuilder;
+use crate::programs::seed::builder::context::SeedContext;
+use crate::programs::seed::builder::course::{CourseBuilder, CourseInstanceConfig};
+use crate::programs::seed::builder::module::{
+    CompletionBuilder, CompletionRegisteredBuilder, ModuleBuilder,
+};
+use crate::programs::seed::builder::page::PageBuilder;
+use crate::programs::seed::seed_courses::CommonCourseData;
+use crate::programs::seed::seed_helpers::paragraph;
+
+use anyhow::{Context, Result};
+use chrono::Utc;
+use headless_lms_models::course_instance_enrollments;
+use headless_lms_models::roles::UserRole;
+use sqlx::Row;
+use tracing::info;
+use uuid::Uuid;
+
+use super::super::seed_users::SeedUsersResult;
+
+async fn get_or_create_default_registrar(conn: &mut sqlx::PgConnection) -> anyhow::Result<Uuid> {
+    if let Some(id) = sqlx::query_scalar::<_, Uuid>(
```

Comment:
Move the database query to the models crate and use the `query_scalar!` macro there.